README.md

```diff
@@ -86,6 +86,7 @@ React Firebase Admin is our in-house admin dashboard boilerplate, used in many o
 - Built-in CD (Continous deployment)
 - PWA ready thanks to CRA and Firebase
 - Multi-tenancy
+- Internationalization (English/Spanish)
 
```

Comment:
I think we should add a new section called internationalization explaining to the user a bit of the folder structure and how it's working in our project. And brief guide on the steps to add a new language into the app :).